build_for_container

```diff
@@ -20,5 +22,5 @@ build_release() {
 }
 
 export GOOS=linux
-export GOARCH=amd64
-build_release x86_64-unknown-linux-gnu
+export GOARCH=$BUILDARCH
+build_release ${BUILDARCH}-unknown-linux-gnu
```

Comment:
This will change the name to `amd64-*` but it doesn't really matter here.  In that case, since we're no longer using a valid GNU host triple, maybe let's drop the `-unknown-linux-gnu` suffix.